docs/static/generated/rest_v1_dispatcher.yml

```diff
@@ -6,7 +6,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: v1/1.17-SNAPSHOT
+  version: v1/1.18-SNAPSHOT
```

Comment:
I think we can split these unrelated change into different commit.

Reply:
ack i will remove it